tests/foreman/destructive/test_ldap_authentication.py

```diff
@@ -33,10 +33,10 @@
 from robottelo.rhsso_utils import create_new_rhsso_user
 from robottelo.rhsso_utils import delete_rhsso_group
 from robottelo.rhsso_utils import delete_rhsso_user
-from robottelo.rhsso_utils import get_rhsso_client_id
 from robottelo.rhsso_utils import run_command
 from robottelo.rhsso_utils import update_rhsso_user
 from robottelo.utils.issue_handlers import is_open
+from robottelo.utils.sso import sso_host
```

Comment:
When this instance is provided via a fixture, the target sat (at the relevant scope) should be passed in.

```
@pytest.fixture
def sso_host(target_sat):
    return SSOHost(satellite=target_sat)
```